index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="assets/css/styles.css">  
+    <title>Document</title>
+</head>
+<body>
+<form action="">
+    
+    <fieldset>
+        <center><h1>Titre</h1></center>2
+        <center><p style="color: yellowgreen;">Apprendre a lire les commandes</p></center>
+       <p> <center> <button ><a href="page2.html">Commencer</a></button>
```

Comment:
for styles and positions do not use html tag. instead, use bootstrap classes or custom css